### DISCUSSION TITLE
Newbie looking for some help re rootkit msgs

### DISCUSSION THREAD

**Discussion by mindfog** [Security-Onion-Solutions/securityonion#2563] [Unsupported Versions]
This is my second install on a second machine where I’m getting rootkit alerts. The machines have not been connected to the network or internet. The monitor card is connected to a DualComm Etap-2205. Install iso passed hash check. Should I be worried about these attached alerts I have many like them including a bunch re ssh policies. Any feedback and would be helpful. Thanks.

![E6E49502-A5D5-4C5F-96EC-BB2810A47730](https://user-images.githubusercontent.com/77118173/103927389-48444300-50e8-11eb-9608-dac0db2089c0.jpeg)
![C7E94BC9-FCAA-414E-99AC-3BBF043BE7A8](https://user-images.githubusercontent.com/77118173/103927396-4a0e0680-50e8-11eb-8651-4945bce61177.jpeg)
![706DCE0B-FCD9-4E96-B7AD-2555CE7914FE](https://user-images.githubusercontent.com/77118173/103927405-4c706080-50e8-11eb-9a41-4ae9808c3a6c.jpeg)
![7E1DBF4B-4F50-4A8E-8119-E084DAA283ED](https://user-images.githubusercontent.com/77118173/103927416-5003e780-50e8-11eb-8091-78aab8848926.jpeg)

**Comment by weslambert** [comment]
You might find solace in the following 😃 : https://github.com/ossec/ossec-hids/issues/1855
